### ISSUE TITLE
neural_tangent.stax network does not produce same outputs as equivalent Keras network

### ISSUE THREAD

**Issue by geajack** [google/neural-tangents#107]
Here's a piece of code that creates a Keras network, uses its weights to initialize an equivalent Stax network, and then runs both of them on the same inputs, comparing the outputs.

```python
import neural_tangents.stax as stax
import jax.random as random
import numpy as np
import keras

key1, key2, key3 = random.split(random.PRNGKey(1), 3)

x = random.normal(key1, (100, 10))

model = keras.models.Sequential(
    [
        keras.layers.Dense(1000, activation="relu"),
        keras.layers.Dense(1, activation="linear")
    ]
)
model.compile()
model.build(input_shape=(1, 10))
weights = model.get_weights()

init_fn, f, _ = stax.serial(
    stax.Dense(1000),
    stax.Relu(),
    stax.Dense(1)
)

_, params = init_fn(key3, x.shape)

params = [
    (weights[0], weights[1]),
    tuple(),
    (weights[2], weights[3])
]

y = f(params, x)

y2 = model.predict(np.array(x))

error = np.linalg.norm(y - y2, 2)
relative_error = error / np.linalg.norm(y)
print(relative_error)
```

This produces a relative error of about 100. **What's odd is that if I modify this code to use `jax.experimental.stax`, it works as expected: I get a relative error of about 10^(-5).** Inspecting the values by hand, it looks as if `neural_tangents.stax` is outputting the same thing as `jax.experimental.stax`, but multiplying all the outputs by 100, as if there's some normalization or scaling factor in the NT code that's not in the original Jax code.

I'm not sure if this is considered a bug, but I would love to at least understand what's going on so I can account for it correctly in my code. I'm trying to get some `neural_tangents` code and some Keras code talking to eachother, so I need to be able to convert my Keras models to equivalent `neural_tangents.stax` models, with the same weights.

**Comment by geajack** [comment]
I figured it out! This is due to `neural_tangent.stax` defaulting to what it calls "NTK parameterization" rather than "standard" parameterization. What this means is that the computation of a neuron's pre-activation (i.e. the output of a Dense layer) is not w.x + b as is typically the case, but rather c w.x + d b, where c and d are certain scalar factors. You can see the code in `neural_tangents.stax` at around line 789.
